src/control/cmd/daos/flags.go

```diff
@@ -17,32 +17,41 @@ import (
 )
 
 /*
-#define D_LOGFAC	DD_FAC(client)
+#include "util.h"
 
-#include <sys/stat.h>
-#include <daos.h>
-#include <daos_fs.h>
+extern int  obj_class_init(void);
+extern void obj_class_fini(void);
 */
 import "C"
 
-type labelOrUUIDFlag struct {
-	UUID  uuid.UUID
-	Label string
+func flagTestInit() (func(), error) {
+	if err := daosError(C.obj_class_init()); err != nil {
+		return nil, err
+	}
+
+	return func() {
+		C.obj_class_fini()
+	}, nil
+}
```

Comment:
Any reason to not put this in the test file?

Reply:
Yes. You can't `import "C"` in test files. You can, however, test code that uses cgo, though. I don't understand the restriction, but it's well-known, e.g. https://github.com/golang/go/issues/4030